src/js/Options/Modules/Options/StoreOptions.svelte

```diff
@@ -49,6 +49,7 @@
     import UserNotesAdapter from "@Content/Modules/UserNotes/UserNotesAdapter";
 
     let settings: Writable<SettingsSchema> = writable(Settings);
+    UserNotesAdapter.getAdapter(); // Set current adapter
```

Comment:
I don't think this is the right way to go around it, basically calling it out of place, only for side effect. There's a better solution, I'll push an edit